systers_portal/systers_portal/settings/base.py

```diff
@@ -164,3 +164,25 @@
 
 # Django Crispy Forms configuration
 CRISPY_TEMPLATE_PACK = 'bootstrap3'
+
+# Settings for Facebook OAuth
+SOCIALACCOUNT_PROVIDERS = \
```

Comment:
This looks good but please don't copy-paste from tutorials. you need to have rationale for each piece of code added. I will not say anything more on this but please remove the permissions/scopes/fields we will not need at all.

Reply:
@SanketDG The piece of settings is from the official documentation provided by allauth. I did not notice the unnecessary fields we will be fetching.My bad.

Reply:
@SanketDG Thank you for pointing that out. I will be careful next time onwards :). Could you please review it again?

Reply:
No worries, also if you are creating test applications for each of the oAuth backends, me, @sakshi1499 and @sammy1997 can use them to test the applications. We can share them privately through maybe email?

Willing to hear other opinions.

Reply:
@SanketDG Sure. If you'll agree on that I will provide the credentials via email and you'll can test that.

Reply:
Works for me @SanketDG @ritwickraj78

Reply:
Same. I think it'll be great @SanketDG